### ISSUE TITLE
Converting circular structure to JSON

### ISSUE THREAD

**Issue by WardBrink** [mendix/MicroflowTimer#14]
Hi,

I got an error in version 4 of this widget. When I added the widget on a pop up page together with the RedirectURL widget, I got a pop up with the following error:

> An error ocurred while executing microflowConverting circular structure to JSON

No further log messages are displayed in my console. The microflow which is called only had a close page action and nothing else. 

When I installed 3.1.1 the issue no longer persisted.

**Comment by Chebychev** [comment]
Hi,

I also get errors when using it in a pop up.
My error is: 
```
MicroflowTimer_widget_MicroflowTimer_4: An error ocurred while executing microflow: 
cyclic object value p/</<@http://localhost:8080/mxclientsystem/mxui/mxui.js?636730511037875781:46:76598.......
```

Changing the navigation layout of the pop up from the PopupLayout_Lightbox to atlas with no other changes makes the problems go away.

With 3.1.1 it works in the pop up

**Comment by Andries-Smit** [comment]
@Chebychev this should be fixed in version V4.0.1. If not please open ticket with support.mendix.com with an simplified test case. Thanks
